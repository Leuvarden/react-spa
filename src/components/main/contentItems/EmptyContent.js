import React from 'react';

export default function EmptyContent () {
    return <div className="content__empty">No films found</div>;
}