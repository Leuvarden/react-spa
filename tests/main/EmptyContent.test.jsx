import React from 'react';
import EmptyContent from './../../src/components/main/EmptyContent.js';

/* eslint-disable no-undef */
describe ('EmptyContent', () => {

    it('renders as expected', () => {
        const empty = shallow(<EmptyContent />);
    
        expect(empty).toMatchSnapshot();
    });

});