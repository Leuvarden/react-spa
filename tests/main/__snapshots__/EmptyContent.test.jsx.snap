// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyContent renders as expected 1`] = `
<div
  class="content__empty"
>
  No films found
</div>
`;
